### PR TITLE
Upgrading to Aurora MySQL version 3 alternative solution to dangling fulltext_index Errors

### PR DESCRIPTION
**Pull request by HSken**
*Description of changes:*

While performing the upgrade we encountered the "dangling fulltext_index" issue described in this documentation.

After reading the Mysql documentation, we decided to first try to OPTIMIZE the tables, this worked and is a far easier and faster solution to solve this issue than the solution provided in this documentation (creating a new table).

Hope this helps people in the future facing the same issue.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: doc_source/AuroraMySQL.mysql80-upgrade-procedure.md [hunk 1]
```diff
@@ -563,7 +563,41 @@ upgrading to avoid compatibility issues."
 
  The next sequence of examples demonstrates how to fix this particular issue and run the upgrade process again\. This time, the upgrade succeeds\. 
 
- First, we go back to the original cluster and create a new table with the same structure and contents as the one with faulty metadata\. In practice, you would probably rename this table back to the original table name after the upgrade\. 
+ First, we go back to the original cluster\. And run the `OPTIMIZE TABLE tbl_name [, tbl_name] ...` on the tables causing the Error: `Table tbl_name contains
+ dangling FULLTEXT index. Kindly recreate the table before upgrade.`.
+ 
+ See Mysql Documentation: [Optimizing InnoDB Full-Text Indexes](https://dev.mysql.com/doc/refman/5.6/en/fulltext-fine-tuning.html#fulltext-optimize) and [OPTIMIZE TABLE Statement](https://dev.mysql.com/doc/refman/5.6/en/optimize-table.html)
```
Comment (claytran) [inline doc_source/AuroraMySQL.mysql80-upgrade-procedure.md]: AWS style is:

For more information, see <reference1> and <reference2> in the MySQL documentation.

Comment (claytran) [reply doc_source/AuroraMySQL.mysql80-upgrade-procedure.md]: the rest of this looks fine- thanks!

**Review by claytran — CHANGES_REQUESTED** [review]
see comments inline

File: doc_source/AuroraMySQL.mysql80-upgrade-procedure.md (hunk 1 above)
Comment (HSken) [reply doc_source/AuroraMySQL.mysql80-upgrade-procedure.md]: Thank you, I changed the sentence.

**Comment by claytran** [comment]
merged- thanks @HSken!

**Pull request merged**